src/backend/src/services/change-request.services.ts

```diff
@@ -0,0 +1,501 @@
+import { ChangeRequest } from 'shared';
+import prisma from '../prisma/prisma';
+import changeRequestRelationArgs from '../prisma-query-args/change-request.query-args';
+import { AccessDeniedException, HttpException, NotFoundException } from '../utils/errors.utils';
+import changeRequestTransformer from '../transformers/change-request.transformer';
+import { Role, CR_Type, WBS_Element_Status } from '@prisma/client';
+import { sendSlackChangeRequestNotification, sendSlackCRReviewedNotification } from '../utils/change-requests.utils';
+import { buildChangeDetail } from '../utils/utils';
+import { getUserFullName } from '../utils/users.utils';
+
+export default class ChangeRequestService {
+  static async getChangeRequestByID(crId: number): Promise<ChangeRequest> {
```

Comment:
each of these should have detailed jsdoc like in the risks service (type /** then press enter and it will set up the template for you)